src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write code here
+
+const body = document.querySelector('tbody');
+const rows = body.querySelectorAll('tr');
+const title = document.querySelector('thead');
+
+title.addEventListener('click', (e) => {
+  const index = e.target.cellIndex;
+  const sorted = Array.from(rows).sort((A, B) => {
+    const cellA = A.children[index].textContent;
+    const cellB = B.children[index].textContent;
+
+    if (index === 3) {
```

Comment:
try to don't use the 'magic number' in the code. If you want to use index 3, you need to decorate it to the cost.